Screens/UserProfileScreen.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import DrawerMenucIcon from "../Navigation/DrawerMenuIcon";
+import LocalizeComponent from "../Localization/LocalizedComponent";
+import {
+    Container,
+    Content,
+    Left,
+    Right,
+    ListItem,
+    Header
+} from "native-base";
+import Text from './Components/CustomText';
+import { FontAwesome } from '@expo/vector-icons'
+
+class UserProfileScreen extends LocalizeComponent {
+    static navigationOptions = {
+        drawerLabel: () => null
+    }
+
+    state = {
+        firstName: "Denys",
+        lastName: "Shourek",
+        email: "test@gmail.com",
+        phone: "+380973122522",
+        birthDate: "1992-11-03T15:27:31.2278615+03:00",
+        city: "Lviv",
+        country: "Ukraine",
+    }
+
+    convertBirthDate() {
```

Comment:
pass date via function parameters - this will allow reuse function in other place